pymsis/meson.build

```diff
@@ -0,0 +1,9 @@
+py3.install_sources([
+  '__init__.py',
+  'msis.py',
+  'msis2.0.parm',
+  'msis21.parm',
+],
+  pure: false,
```

Comment:
... renders this obsolete, and impossible to forget in future `py3.install_sources()` calls.


```suggestion
```